test/nn/models/test_basic_gnn.py

```
@@ -276,17 +278,27 @@ def forward(self, x, edge_index):
     assert expected.size() == (3, 16)
 
     path = osp.join(tmp_path, 'model.onnx')
-    torch.onnx.export(
+    success = safe_onnx_export(
         model,
         (x, edge_index),
         path,
         input_names=('x', 'edge_index'),
         opset_version=18,
         dynamo=True,  # False is deprecated by PyTorch
+        skip_on_error=True,  # Skip gracefully in CI if upstream issue occurs
```

@AJamal27891 @puririshi98 Mind reminding us of why the failure is now skipped? Doesn't it just hide the current failure (and also future errors)?

---

> 1. "Encountered known ONNX serialization issue (SerdeError). This is likely the allowzero boolean attribute bug. Attempting workaround..."

Also, where can I find a GitHub issue for the "bug"?

---

> > 1. "Encountered known ONNX serialization issue (SerdeError). This is likely the allowzero boolean attribute bug. Attempting workaround..."
> 
> Also, where can I find a GitHub issue for the "bug"?
I raised an issue here 
https://github.com/pytorch/pytorch/issues/161941

---

@AJamal27891 I meant even if it's opt-in, we shouldn't just ignore the error in our CI. Instead, we should:
* report it to the PyTorch/ONNX team (which you did, and thank you!), and
* let it fail and skip it in our CI by noting that it's an issue with ONNX (preferably with a tracking issue)

I also skimmed thorough the references you kindly provided, but it's hard for me to understand how they're helpful or relevant here.